surface/src/on_nurbs/fitting_surface_pdm.cpp

```diff
@@ -38,6 +38,7 @@
 #include <pcl/surface/on_nurbs/fitting_surface_pdm.h>
 #include <pcl/pcl_macros.h>
 
+#include <Eigen/Cholesky>
```

Comment:
Please add the `// for` recommended by IWYU in additions

Reply:
Added, thanks.